mne_bids/read.py

```diff
@@ -20,42 +20,42 @@
 from .tsv_handler import _from_tsv, _drop
 from .config import ALLOWED_EXTENSIONS
 from .utils import (_parse_bids_filename, _extract_landmarks,
-                    _find_matching_sidecar, _parse_ext)
+                    _find_matching_sidecar, _parse_ext, _get_ch_type_mapping)
 
 reader = {'.con': io.read_raw_kit, '.sqd': io.read_raw_kit,
           '.fif': io.read_raw_fif, '.pdf': io.read_raw_bti,
           '.ds': io.read_raw_ctf, '.vhdr': io.read_raw_brainvision,
-          '.edf': io.read_raw_edf, '.bdf': io.read_raw_edf,
+          '.edf': io.read_raw_edf, '.bdf': io.read_raw_bdf,
           '.set': io.read_raw_eeglab}
 
 
-def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
+def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
               montage=None, verbose=None, allow_maxshield=False):
     """Read a raw file into MNE, making inferences based on extension."""
-    fname, ext = _parse_ext(raw_fname)
```

Comment:
what is the difference between fpath and fname for you @sappelhoff ? We tend to use everywhere fname for any string that points to a file on disk

Reply:
for me this is an `fpath`

> /home/stefanappelhoff/Desktop/foo.py

and this is an `fname`

> foo.py

I also don't use it consistently ... but when I changed it in this PR, it was so that variables do not get overridden, when I need the old values.